operators/operator_sdk/osdk-getting-started.adoc

```diff
@@ -16,7 +16,7 @@ and the Operator Lifecycle Manager (OLM).
 
 [NOTE]
 ====
-{product-title} 4.3 supports Operator SDK v0.12.0 or later.
+{product-title} 4.4 supports Operator SDK v0.12.0 or later.
```

Comment:
:+1: Getting this here too but will wait for this to merge first: https://github.com/openshift/openshift-docs/pull/21467